app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/contactname/contact_name.scala.html

```diff
@@ -31,6 +31,7 @@
 
 
 @(form: Form[ContactName],
+        amendingContactName: Boolean,
```

Comment:
do we actually need this in this template? There's a similar thing for other templates because the content changes depending on whether or not you're entering it in for the first time or you're amending what you've already given us but I don' think that's the case here.

Reply:
I added for the banner to indicate to the user there has been a change. Or are we using another mechanism?

Reply:
the banner would appear on the homepage rather than this one - once the redirect to the homepage is done after submitting a contact name then the idea is for the banner to displayed on the homepage

Reply:
fixed